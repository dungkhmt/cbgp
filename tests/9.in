50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
