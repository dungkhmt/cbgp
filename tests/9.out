(31, 48), (42, 38), (45, 23), (23, 47), (42, 33), (44, 7), (33, 3), (32, 43), (27, 3), (35, 6), (35, 12), (16, 4), (45, 10), (35, 8), (41, 5), (14, 7), (19, 0), (3, 4), (3, 5), (7, 4), (17, 16), (34, 10), (11, 16), (46, 10), (37, 19), (44, 13), (48, 3), (40, 45), (29, 20), (5, 1), (30, 0), (21, 15), (19, 26), (7, 8), (22, 16), (30, 11), (28, 23), (12, 25), (47, 38), (28, 2), (24, 18), (21, 1), (29, 18), (11, 43), (44, 9), (39, 15), (30, 20), (36, 11), (0, 1), (44, 2), (13, 2), (48, 1), (41, 21), (34, 13), (14, 8), (32, 16), (4, 33), (41, 15), (12, 0), (7, 0), (13, 10), (41, 26), (44, 6), (5, 0), (28, 10), (29, 24), (22, 36), (38, 31), (31, 3), (20, 18), (17, 22), (30, 18), (45, 28), (1, 2), (22, 11), (25, 6), (42, 31), (47, 10), (12, 8), (41, 19), (25, 2), (34, 46), (8, 6), (40, 47), (44, 8), (32, 30), (38, 1), (32, 7), (32, 11), (12, 6), (48, 38), (37, 41), (45, 47), (31, 33), 
0: (42, 13), 1: (22, 5), 2: (20, 6), 3: (52, 3), 4: (74, 4), 5: (49, 6), 6: (29, 15), 7: (48, 30), 8: (35, 17), 9: (32, 18), 10: (10, 10), 11: (68, 8), 12: (38, 14), 13: (15, 11), 14: (37, 18), 15: (40, 8), 16: (72, 5), 17: (65, 5), 18: (57, 8), 19: (45, 9), 20: (60, 5), 21: (37, 7), 22: (66, 6), 23: (10, 6), 24: (57, 7), 25: (28, 14), 26: (46, 8), 27: (54, 5), 28: (14, 6), 29: (55, 6), 30: (51, 22), 31: (51, 2), 32: (53, 24), 33: (78, 0), 34: (14, 12), 35: (35, 16), 36: (68, 7), 37: (44, 10), 38: (15, 0), 39: (39, 9), 40: (10, 9), 41: (41, 7), 42: (51, 0), 43: (53, 23), 44: (41, 37), 45: (11, 8), 46: (13, 13), 47: (0, 0), 48: (23, 4), 49: (34, 18), 
42 13 22 5 20 6 52 3 74 4 49 6 29 15 48 30 35 17 32 18 10 10 68 8 38 14 15 11 37 18 40 8 72 5 65 5 57 8 45 9 60 5 37 7 66 6 10 6 57 7 28 14 46 8 54 5 14 6 55 6 51 22 51 2 53 24 78 0 14 12 35 16 68 7 44 10 15 0 39 9 10 9 41 7 51 0 53 23 41 37 11 8 13 13 0 0 23 4 34 18 
intersectionCount=0
minAngle=0
sumAngle=1e+18
minEdgeLength=1e+18
sumEdgeLength=0
minDistance=1e+18
sumDistance=0

0: (42, 13)
1: (22, 5)
2: (20, 6)
3: (52, 3)
4: (74, 4)
5: (49, 6)
6: (29, 15)
7: (48, 30)
8: (35, 17)
9: (32, 18)
10: (10, 10)
11: (68, 8)
12: (38, 14)
13: (15, 11)
14: (37, 18)
15: (40, 8)
16: (72, 5)
17: (65, 5)
18: (57, 8)
19: (45, 9)
20: (60, 5)
21: (37, 7)
22: (66, 6)
23: (10, 6)
24: (57, 7)
25: (28, 14)
26: (46, 8)
27: (54, 5)
28: (14, 6)
29: (55, 6)
30: (51, 22)
31: (51, 2)
32: (53, 24)
33: (78, 0)
34: (14, 12)
35: (35, 16)
36: (68, 7)
37: (44, 10)
38: (15, 0)
39: (39, 9)
40: (10, 9)
41: (41, 7)
42: (51, 0)
43: (53, 23)
44: (41, 37)
45: (11, 8)
46: (13, 13)
47: (0, 0)
48: (23, 4)
49: (34, 18)

Time taken: 0.022741 s
